### DISCUSSION TITLE
problem importing .asreview file back into asreview for sequential screening

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1668] [Q&A]
Dear ASReview community, 

I exported a current asreview project as an .asreview file from the export option "project including data and model configuration" (see screenshot below), and I wanted to import this .asreview file back into asreview to continue screening. I would like to do this to enable sequential screening between multiple screeners who import and export the file and can thereby continue where the other person stopped.

However, I am not able to import the .asreview file again. When I make a new project, select oracle mode, and try to then import the .asreview file it does not show me this file as an option, although it is in the selected folder. I am sure that I am making some silly mistake, but I have looked around in the discussions and have not found an answer yet. 

Thank you a lot in advance for any help you can provide! :) 

Best wishes!

<img width="667" alt="asreview lab screenshot" src="https://github.com/asreview/asreview/assets/157508150/ee85c25f-6da5-46da-b683-cbee1e08cd10">

**Comment by Rensvandeschoot** [comment]
Ah, yes. I see the issue. You have selected creating a new project and then you can only import a dataset. You need to click the Import project icon on the top right in the dashboard; See, for example, this screenshot in the documentation https://asreview.readthedocs.io/en/latest/manage.html. Hope this helped!!

**Reply by Morle2104** [reply]
Ah, perfect! Thanks so much for the quick and helpful reply!
